proofread installation chapter (#2374)
centreon/centreon-documentation: versioned_docs/version-23.10/installation/installation-of-a-central-server/using-sources.md

/bin/pear channel-update pear.php.net
```

If you can’t access the Internet directly but have to go via a proxy,

> Use the following command:

Reviewer: ```suggestion
use the following command:
```

/bin/sed                                                   OK

------------------------------------------------------------------------

>         Check the mandatory gorgone service status

Reviewer: ```suggestion
        Check mandatory gorgone service status
```

### License agreement

``` shell

> This General Public License does not permit you to incorporate your program into

Reviewer: This General Public License does not permit incorporating your program into

> This General Public License does not permit you to incorporate your program into

Reviewer: ```suggestion
This General Public License does not permit incorporating your program into
```

proprietary programs.  If your program is a subroutine library, you may
consider it more useful to permit linking proprietary applications with the
library.  If this is what you want to do, use the GNU Library General
Public License instead of this License.

> Do you accept the GPLv2 license?

Reviewer: ```suggestion
Do you accept GPLv2 license ?
```
